.github/workflows/cygwin_build.yml

```diff
@@ -13,21 +13,65 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: 'recursive'
+      - name: Prepare ccache timestamp
+        id: ccache_cache_timestamp
+        shell: bash
+        run: |
+          NOW=$(date -u +"%F-%T")
+          echo "timestamp=${NOW}" >> $GITHUB_OUTPUT
+          WORKFLOWNAME="${{github.workflow}}"
+          NAME_DASHED=${WORKFLOWNAME//+( )/_}
+          echo "cache-key=${NAME_DASHED}" >> $GITHUB_OUTPUT
+
       - uses: cygwin/cygwin-install-action@master
         with:
-          packages: cygwin64 gcc-g++=10.2.0-1 python37 python37-future python37-lxml python37-pip python37-setuptools python37-wheel git procps gettext
+          packages: cygwin64 gcc-g++=10.2.0-1 ccache python37 python37-future python37-lxml python37-pip python37-setuptools python37-wheel git procps gettext
+          add-to-path: false
+      # Put ccache into github cache for faster build
+      - name: setup ccache
+        env:
+          PATH: /usr/bin:$(cygpath ${SYSTEMROOT})/system32
+        shell: C:\cygwin\bin\bash.exe -eo pipefail '{0}'
+        run: >-
+          mkdir -p /cygdrive/d/a/ardupilot/ardupilot/ccache &&
+          mkdir -p /usr/local/etc &&
```

Comment:
What's this for?

Reply:
leftover... should be /usr/local/bin/ now